### DISCUSSION TITLE
Website not appreaing in google searches

### DISCUSSION THREAD

**Discussion by varadVaidya** [alshedivat/al-folio#1314] [Q&A]
Hello all,
I have created a website with this template. I have also added the google analytics and google console keys, and have verified that the analytics is working. 
But still even after a week, my website is not appearing in Google Search. It DOES appear in a Bing search, so it seems that my page hasn't been indexed by Google yet. 
So my question is what steps I can do to get the website in Google searches as well, other than just waiting and praying that it indexes soon ?

**Comment by ulvinwalex789** [comment]
If your website is not appearing in Google searches, it could be due to a variety of reasons, including poor search engine optimization (SEO) practices, technical issues, or low-quality content. To improve your website's visibility in Google searches, you can start by optimizing your website for relevant keywords and ensuring that your website is mobile-friendly, fast-loading, and has a good user experience. You can also consider building high-quality backlinks from reputable websites and creating engaging content that resonates with your target audience.

As for [Sportzfy Apk ](https://pikashows.net/sportzfy/)website, it's important to note that Google's algorithms are constantly changing, and there are no guarantees that a website will always appear in the search results. However, there are some steps you can take to increase your website's chances of appearing in Google searches. These include optimizing your website for relevant keywords, creating high-quality content, and building high-quality backlinks from reputable websites. Additionally, you can use Google Search Console to monitor your website's performance in Google searches and identify any issues that may be affecting its visibility.

**Comment by georgemilosh** [comment]
I have similar problem. I tried using Google console. And it complained: 
```
Duplicate without user-selected canonical
```
I did some google searches around this topic and it seems i have to modify <head> to include canonical link to my main webpage. I don't know how to do that
